ubuntu/ubuntu-18.04-hpc/install_nvidiagpudriver.sh

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
 set -ex
 
-# Change versions (Ubuntu, GPU driver) here if needed
-# See https://github.com/Azure/azhpc-extensions/blob/master/NvidiaGPU/resources.json
-DRIVER_URL=https://go.microsoft.com/fwlink/?linkid=874271
-CUDA_REPO_PKG=cuda-repo-ubuntu_amd64.deb
-PUBKEY_URL=http://download.microsoft.com/download/F/F/A/FFAC979D-AD9C-4684-A6CE-C92BB9372A3B/7fa2af80.pub
+wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-ubuntu1804.pin
+mv cuda-ubuntu1804.pin /etc/apt/preferences.d/cuda-repository-pin-600
```

Comment:
We may want to unify this with how extensions are installed.

Reply:
Yes, as I mentioned in the seed PR, the end objective is to unify the extension with the scripts here. I can talk about how that can be achieved. But basically that encompasses not having multiple scripts in different places doing the same thing.
As a first pass and PoC on the Ubuntu image, I am fine with us not tackling that issue yet.

However, why should we remove the variables? Also, the "pin" method above is different than manual steps we publish and extension. Any advantage with this?